### PR TITLE
Add receivabl.es to Payments

### PR DESCRIPTION
**Pull request by tehpeh**

### CODE REVIEW

**Review by mxxcon — CHANGES_REQUESTED** [review]
@tehpeh, according to your exception text that makes this two-factor _authorization_, not authentication, which means it should be `tfa: no` and other relevant fields updated as well

**Comment by tehpeh** [comment]
Ok, let me rephrase: payment senders are required to enter their email address and a one-time SMS code sent to their phone to access their saved personal data, conversely payment receivers are authenticated via OAuth with Xero (which supports TFA). Does that count?

**Comment by mxxcon** [comment]
@tehpeh See https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#a-note-on-definitions if that helps you clarify things.

**Comment by tehpeh** [comment]
Ok I'm not sure now, because if your personal information is stored on the site, you are prompted to enter a token during login. But this only applies to half the user base (ie. the payment senders).

**Comment by mxxcon** [comment]
How does it know that today I want to send vs receive a payment? Are those completely different accounts?

**Comment by tehpeh** [comment]
Yes, completely different accounts. I can email you a link to a demo if you'd like?

**Comment by mxxcon** [comment]
Is there a publicly reachable docs link you provide?
At the very least the exception text would need to rephrased clarifying that only payment accounts have tfa, not recipients.

**Comment by tehpeh** [comment]
No documentation yet, but I'll send an update as soon as it's available.

**Comment by Jawshy** [comment]
Hello, @tehpeh. Would you be able to provide an update on the site's documentation of how it has implemented two-factor auth?

**Comment by tehpeh** [comment]
Still no documentation, but this PR can be closed now thanks.
